scripts/ci/test-operator

```diff
@@ -118,10 +118,20 @@ trap_add_exit "log_operator_state $LOG_FILE OBJECTS $NAMESPACE; cat $LOG_FILE"
 # Wait for the deployment specified in the CSV to rollout successfully.
 wait_on_deployment "$DEP_NAME" "$NAMESPACE"
 
+sleep 180
+
+kubectl get pods -n $NAMESPACE
+
+trap_add_exit "kubectl logs -lcomponent=api-server --all-containers=true -n $NAMESPACE"
```

Comment:
I'm not familiar with `--all-containers`, could this work with getting logs for all containers in a namespace, or does it require a selector as in this case?
I think this line and the line after are a bit too specific to this operator. I'm working on #240 to add more logging after test completion, which includes getting OLM logs and logs from all pods/containers in the operator namespace, so I'm wondering if I can reuse some of this there.

Reply:
It was new to me as well, but another approach is a generalized version of this script: https://github.com/EnMasseProject/enmasse/blob/master/systemtests/scripts/collect_logs.sh

I will remove this and other parts specific to our operator.